default.go

```diff
@@ -84,7 +87,7 @@ func (d *DefaultNode) setDefaults(fields models.Fields, tags models.Tags) (model
 	newTags := tags
 	tagsCopied := false
 	for tag, value := range d.d.Tags {
-		if _, ok := tags[tag]; !ok {
+		if v := tags[tag]; v == "" {
```

Comment:
Ah, I see this is the breaking change. I guess we should break things :) Should we enable this behavior behind a flag? Something like 
```
|default()
   .tag('dc', 'sfc')
   .defaultEmptyTags()
```